### DISCUSSION TITLE
Dozens of event_messages "password authentication failed for user "postgres""

### DISCUSSION THREAD

**Discussion by AlexanderCryptoOwl** [supabase/supabase#5854] [Questions] [answered]
Hi,

I just saw a lot of these in my DB logs. Looks dodgy to me, I wonder does Supabase provide some level of security around spam like this? Like block auth for that user for X amount of time or the IP

<img width="673" alt="image" src="https://user-images.githubusercontent.com/99205909/157211146-6664505b-e36d-4437-a386-c9f516b6d87c.png">

**Comment by GaryAustin1** [comment, marked as answer]
I believe those are coming from your postgres connection, not auth for users.  
SB has fail2ban on that port.  
If you have anymore of them you might see if the ip address is the same in the details. 

https://supabase.com/blog/2021/07/26/supabase-postgres-13  mentions 3 times should be 10 minute ban, but not sure that is latest setup.

**Comment by rr404** [comment]
Hey, 
There is also the CrowdSec solution: https://www.crowdsec.net/blog/hardening-self-hosted-supabase
The advantage with CrowdSec is that if you're part of the network, you also receive the community blocklist.
And there are more attack and misbehavior scenarios coming soon.
